sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -488,10 +488,24 @@ def to_bigquery(
             return str(job_config.destination)
 
         with self._query_generator() as query:
-            self._execute_query(query, job_config, timeout)
+            dest = job_config.destination
+            # because setting destination for scripts is not valid
```

Comment:
was this just not working before? Seems like it was using the job_config.destination before?

Reply:
when we run multiple scripts separated by semicolon, BigQuery just doesn't accept passing destination explicitly, will throw error
`google.api_core.exceptions.BadRequest: 400 configuration.query.destinationTable cannot be set for scripts`

that's why I add some workaround here

Reply:
simply retrieve BQ-generated temporary table after scripts being executed (should look like this `<project_id>._08dc845331ibb01824ea0f3c3e079e93b3a076.anon933abd7cb6950c3c488af61fe94ad1713b6820cd1115fddd0c09cde32ec12823` )

then we persist it as our naming convention with prefix `historical_`